vnext/template/cpp-app/proj/MyApp.vcxproj

```diff
@@ -26,6 +21,9 @@
     {{/certificateThumbprint}}
     <PackageCertificatePassword>password</PackageCertificatePassword>
   </PropertyGroup>
+  <PropertyGroup Label="NuGet">
+    <ResolveNuGetPackages>false</ResolveNuGetPackages>
```

Comment:
Curious why this setting is here, seen it in a bunch of proejcts...

Reply:
In short, makes the `Restore` target work on native projects. Will fail complaining about frameworks otherwise.
I don't know the core explanation, though.